app/test/square_size_test.go

```diff
@@ -73,34 +75,38 @@ func (s *SquareSizeIntegrationTest) TestSquareSizeUpperBound() {
 	}
 
 	tests := []test{
-		{
-			name:             "default",
-			govMaxSquareSize: appconsts.DefaultGovMaxSquareSize,
-			maxBytes:         appconsts.DefaultMaxBytes,
-			expMaxSquareSize: appconsts.DefaultGovMaxSquareSize,
-		},
 		{
 			name:             "max bytes constrains square size",
-			govMaxSquareSize: appconsts.DefaultGovMaxSquareSize,
+			govMaxSquareSize: 64,
 			maxBytes:         appconsts.DefaultMaxBytes,
-			expMaxSquareSize: appconsts.DefaultGovMaxSquareSize,
-		},
-		{
-			name:             "gov square size == hardcoded max",
-			govMaxSquareSize: appconsts.SquareSizeUpperBound,
-			maxBytes:         appconsts.DefaultUpperBoundMaxBytes,
-			expMaxSquareSize: appconsts.SquareSizeUpperBound,
+			expMaxSquareSize: 64,
 		},
```

Comment:
see op comment, felt this test was redundant and outdated a bit now that we have huge blonks. we should flup by changing to one or more unit tests